src/assets/crisistextline/sass/partials/01-base/_colors.scss

```diff
@@ -42,10 +42,10 @@ $color-todo: $color-green;
 
 // CC CONVO TITLEBAR COLORS
 $color-0:           $color-blue;
-$color-1:           $color-notification;
+$color-1:           #2a69b2;    // dark cyan AKA lochmara
```

Comment:
just a small nit, but can you make the letters caps here, in line w/ the rest of the colors?